ntoskrnl/include/internal/mm.h

```diff
@@ -1282,7 +1290,11 @@ MmFindRegion(
 #define IS_DIRTY_SSE(E)          ((E) & 2)
 #define WRITE_SSE(E)             ((E) | 4)
 #define IS_WRITE_SSE(E)          ((E) & 4)
+#ifdef _WIN64
```

Comment:
```suggestion
#ifdef _M_AMD64
```

Reply:
Well the whole SSE stuff is horrible, but it's generally not architecture dependent, except for 32 vs 64 bits, so using _WIN64 is intentionally here. Should work with any 64 bit system.